lib/std/Target.zig

```diff
@@ -1163,6 +1168,8 @@ pub const Cpu = struct {
         x86_64,
         xcore,
         xtensa,
+        propeller1,
+        propeller2,
```

Comment:
Please keep the list sorted.

Reply:
I checked if it was sorted and it yielded a different result as it was before, so i assumed it was not sorted. I'll update the changes :)

Reply:
Well, uh, it's *sorta* sorted... I think the only exception is that `thumb` is after `arm` and `aarch64` is after both, mainly to group them logically.

Reply:
Okay, this should be resolved now.